antora/ec-policies-antora-extension/index.js

```diff
@@ -114,6 +114,7 @@ const helpers = {
           const title = a.annotations.title
           const description = a.annotations.description
           const shortName = a.annotations.custom.short_name
+          const anchor = `${a.path[2].value}_${a.path[3].value}_${shortName}`
```

Comment:
Does `fullPath` hold pretty much the same value?

Reply:
It has dots and no `shortName` in it, so not the best option...

Reply:
I was thinking `${packagePath}_${shortName}` might, or `${packagePath.replaceAll('.', '_')}_${shortName}`.